pages/1.12/installing/production/system-requirements/index.md

```diff
@@ -218,23 +218,47 @@ sudo yum install -y tar xz unzip curl ipset
 
 ### Cluster permissions (production installation)
 
-On each of your cluster nodes, use the following command to:
+On each of your cluster nodes, follow the below instructions:
 
-*   Disable SELinux or set it to permissive mode.
-*   Add `nogroup` and `docker` to each of your Mesos masters and agents.
-*   Reboot your cluster for the changes to take effect.
+*   Make sure that SELinux is in one of the supported modes.
+
+    To review the current SELinux status and configuration run the following command:
+
+    ```bash
+    sudo sestatus
+    ```
+
+    DC/OS supports the following SELinux configurations:
+
+    * Current mode: `disabled`
+    * Current mode: `permissive`
+    * Current mode: `enforcing`, given that `Loaded policy name` is `targeted`
```

Comment:
Here we should clarify that we do not support when SELinux monitors DC/OS.

We could add "but not targeted at DC/OS".